client/session_test.go

```diff
@@ -94,7 +94,7 @@ func TestClient_AutoRenew_Goroutine(t *testing.T) {
 		if err != nil {
 			t.Errorf("could not get client's session: %v", err)
 		}
-		if time.Now().UTC().After(endTime) {
```

Comment:
I've seen this failing, eventually, because of the precision, which isn't considered when manipulating session's end time. 

I don't know why I didn't fail before, or perhaps it was just a flaky test 🤔